documentation/modules/creating-validation-rule.adoc

```diff
@@ -121,7 +121,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
-<3> Specify the name of provider `Secret` CR.
-
+<2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<{rhv-short}_engine_host>/ovirt-engine/api/` for {rhv-short}.
+<3> Specify the name of the provider `Secret` CR. 
++
```

Comment:
@RichardHoch I don't know what is the effect of this addition (+), do we need it?

Reply:
Was not needed; removed

Reply:
Removed.